posthog/migrations/0926_cohortcalculationhistory_cohort_started_idx.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.postgres.operations import AddIndexConcurrently
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add index to speed up retrieval of the most recent calculation result per cohort.
+    Without this index, each subquery requires an in-memory sort (~0.9ms per cohort).
+    With ~500 cohorts, that adds ~450ms to the cohort list API response.
+    """
+
+    atomic = False
+
+    dependencies = [
+        ("posthog", "0925_team_business_model"),
+    ]
+
+    operations = [
+        AddIndexConcurrently(
```

Comment:
Minor nit, but this index will make the existing `posthog_cohortcalculationhistory_cohort_id_e7c02b55` index on `cohort_id` redundant. Will you be removing that one in a follow-up migration?

Reply:
Good point, I'll do that